pkg/injection/namespace/bootstrapperconfig/bootstrapperconfig.go

```diff
@@ -88,11 +104,37 @@ func (s *SecretGenerator) GenerateForDynakube(ctx context.Context, dk *dynakube.
 		if err != nil {
 			return errors.WithStack(err)
 		}
+
+		// create cert secret for otlp configuration
+		// TODO is the creation of the OTLP related secrets tied to the same preconditions as the creation of the bootstrap secrets?
+		// TODO should we decouple the OTLP secret creation to be independent of any errors during the creation of the other secrets?
```

Comment:
I would be for a separate otlp secret, created by a separate package

The whole idea behind the `bootstarpperconfig` is that it creates a secret with all the necessary config files for the injected init-container, the webhook doesn't read it really, just ensures the existence of the secret (as best it can) and mounts it to the ini-container

the interesting thing is the certs only secret, the maybe could be reused, depending on how you will use it